### PR TITLE
NXP-24346: Adding to elasticsearch pageproviders

### PR DESCRIPTION
**Pull request by covolution**

### CODE REVIEW

File: nuxeo-distribution/nuxeo-server-tomcat/src/main/resources/tomcat/bin/nuxeo.conf [hunk 1]
```diff
@@ -108,7 +108,7 @@ elasticsearch.httpEnabled=true
 
 ## Comma separated list of CorePageProvider to supersede by Elasticsearch.
 ##  default is default_search,DEFAULT_DOCUMENT_SUGGESTION.
-#elasticsearch.override.pageproviders=default_search,document_content,section_content,document_content,tree_children,default_document_suggestion,simple_search,advanced_search,nxql_search,DEFAULT_DOCUMENT_SUGGESTION
+#elasticsearch.override.pageproviders=default_search,section_content,document_content,tree_children,default_document_suggestion,simple_search,advanced_search,nxql_search,DEFAULT_DOCUMENT_SUGGESTION,REST_API_SEARCH_ADAPTER,all_collections
```
Comment (efge) [inline nuxeo-distribution/nuxeo-server-tomcat/src/main/resources/tomcat/bin/nuxeo.conf]: Here I would not try to maintain a list of possible options. And the mention of the default is actually wrong. So maybe just write:

    ## Comma separated list of CorePageProvider to supersede by Elasticsearch.
    ## The default is defined by nuxeo.defaults in template common-base
    #elasticsearch.override.pageproviders=...

Comment (bdelbosc) [reply nuxeo-distribution/nuxeo-server-tomcat/src/main/resources/tomcat/bin/nuxeo.conf]: Or even simpler remove these lines.

**Comment by efge** [comment]
👍

**Comment by driverpt** [comment]
👍

**Pull request merged**